templates/taxi/driver_list.html

```diff
@@ -6,8 +6,14 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver == request.user %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }}) (Me)
+          {% else %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }})
+          {% endif %}
```

Comment:
You also can do this in different way (not to duplicate the same code)
```suggestion
          
            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
            ({{ driver.first_name }} {{ driver.last_name }}) {% if driver == request.user %}(Me){% endif %}
          
```

Reply:
Thank you!